pages/pil/dashboard/index.js

```diff
@@ -1,12 +1,39 @@
 const page = require('../../../lib/page');
 const modulesToCertificates = require('../../../lib/utils/modules-to-certificates');
+const bodyParser = require('body-parser');
 
 module.exports = settings => {
   const app = page({
     ...settings,
     root: __dirname
   });
 
+  const profileOwnsModules = (profile, modules) => modules.every(
+    moduleId => !!profile.trainingModules.find(module => module.id === moduleId)
+  );
```

Comment:
Could we do something like this to reduce the number of nested functions inside functions?

```js
const profileOwnsModules = (profile, modules) => {
  const userModules = profile.trainingModules.map(m => m.id);
  return modules.every(id => userModules.includes(id));
});
```